src/pages/ZNS/ZNS.tsx

```diff
@@ -61,7 +61,7 @@ const ZNS: React.FC<ZNSProps> = () => {
 	const isMobile = useMatchMedia('phone');
 	const isMobilePortrait = useMatchMedia('(max-width: 569px)');
 
-	const enableBanner = false;
+	const enableBanner = true;
```

Comment:
enabled current banner and removed out dated content from this branch. 

As discussed we could title the current banner as v1 and another as v2